centreon/www/front_src/.gitignore

```diff
@@ -19,4 +19,8 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-junit.xml
\ No newline at end of file
+junit.xml
+
+# private
+/src/**/__tests__/*.http
+*.private.env.json
```

Comment:
```suggestion
```

Reply:
fyi. keeping these till navigation re-factor (which will touch the `/api`)